src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/casenotes/CaseNotesReconciliationService.kt

```diff
@@ -226,23 +226,24 @@ data class MismatchCaseNote(
   val offenderNo: String,
   val diffsForDps: Set<String> = emptySet(),
   val diffsForNomis: Set<Long> = emptySet(),
-  var notes: List<String> = emptyList(),
+  val notes: List<String> = emptyList(),
 )
 
-data class CommonCaseNoteFields(
+data class ComparisonCaseNote(
```

Comment:
renamed as these weren't really common fields before I added the ids 😁.  It is more a class that is used to compare two case notes